nix/sources.json

```diff
@@ -5,10 +5,10 @@
         "homepage": null,
         "owner": "clash-lang",
         "repo": "ghc-tcplugins-extra",
-        "rev": "380e4510ae94e6b6cfb1bcacf633437e2c7226a4",
-        "sha256": "0fwfjvqw4k9lk23xrysiw54khfcif2s2cg52zwkgd0wbhjg6rrmd",
+        "rev": "684f3aa53c193220974c542ed337ec94e5265291",
+        "sha256": "19sczl14hcaai20ckrd8gwqbrzyaby9pyv488ljxizq4ynqdx380",
         "type": "tarball",
-        "url": "https://github.com/clash-lang/ghc-tcplugins-extra/archive/380e4510ae94e6b6cfb1bcacf633437e2c7226a4.tar.gz",
+        "url": "https://github.com/clash-lang/ghc-tcplugins-extra/archive/684f3aa53c193220974c542ed337ec94e5265291.tar.gz",
```

Comment:
If you didn't already; niv can help you do this automatically: https://hackage.haskell.org/package/niv

Reply:
Thanks! I didn't, so I had to do this manually.